internal/cmd/grpc.go

```diff
@@ -279,11 +279,29 @@ func NewGRPCServer(
 	skipAuthIfExcluded(metasrv, cfg.Authentication.Exclude.Metadata)
 	skipAuthIfExcluded(evalsrv, cfg.Authentication.Exclude.Evaluation)
 
+	var checker *audit.Checker
+
+	// We have to check if audit logging is enabled here for informing the authentication service that
+	// the user would like to receive token:deleted events.
+	if cfg.Audit.Enabled() {
+		var err error
+		checker, err = audit.NewChecker(cfg.Audit.Events)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	var tokenDeletedEnabled bool
+	if checker != nil {
+		tokenDeletedEnabled = checker.Check("token:deleted")
```

Comment:
does this mean that we arent logging events for `token:created`? I remember talking about this when we initially worked on audit events. I guess its because the `token:created` can be a result of someone logging in to Flipt via OIDC/GitHub and not an explicit creating an API token?

Reply:
So we are logging events for `token:created` but it seems that it is just for static tokens: https://github.com/flipt-io/flipt/blob/main/internal/server/middleware/grpc/middleware.go#L396-L397

Hmm you are right about it not being for the case for OIDC/GitHub logins.

Reply:
makes sense, i was just wondering why we were checking for `token:deleted` only here